index.js

```diff
@@ -17,6 +17,45 @@ function relative(a, b) {
   return relativePath.charAt(0) !== '.' ? './' + relativePath : relativePath;
 }
 
+/*
+ * Checks if there is already a prepend in the current match.
+ */
+function alreadyHasPrepend(string, prepend, offset, submatch) {
```

Comment:
perhaps use `function alreadyHasPrepend(string, prepend, offset, submatch = '') {` to avoid ugly `(submatch || "").length`

Reply:
I wish. Is node >= 0.10 :(